sha1-validation/test.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "sha1-validation",
+    "enabled": true,
+    "requiresSdk": true,
+    "version": "6.0",
+    "versionSpecific": false,
+    "type": "bash",
+    "cleanup": true,
+    "skipWhen": [
+    ],
+    "ignoredRIDs":[
+        "alpine",
+        "fedora",
+        "centos",
+        "rhel7",
+        "rhel8"
```

Comment:
Taking a step back, the original customer report was that `HttpClientHandler` + `CheckCertificateRevocationList` fails against https://redhat.com. They told us that it works on RHEL 8 but not 9, which is why we are looking at a fix in RHEL 9. It seems to me that we want this test to run on all these platforms too, to verify that those platforms are not affected by the same issue that we confirmed exists on RHEL 9.

Reply:
I'll amend it.